lib/ansible/cli/vault.py

```diff
@@ -384,6 +384,8 @@ def execute_encrypt_string(self):
                 sys.stderr.write(err)
             b_outs.append(to_bytes(out))
 
+        # Output must end with a newline to prevent bug #78932
```

Comment:
I don't think that it's wise to leave the most relevant bits of the motivation in some external resource. Let's come up with something that can provide insight even offline. External references are useful for extra details.

Like this:
```suggestion
        # The output must end with a newline to play nice with terminal representation.
        # Refs:
        # * https://stackoverflow.com/a/729795/595220
        # * https://github.com/ansible/ansible/issues/78932
```